bundles/core/org.eclipse.smarthome.core.transform.test/src/test/java/org/eclipse/smarthome/core/transform/internal/JSonPathTransformationServiceTest.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.transform.internal;
+
+import org.eclipse.smarthome.core.transform.TransformationException;
+import org.eclipse.smarthome.core.transform.internal.service.JSonPathTransformationService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Gaël L'hopital
+ */
+public class JSonPathTransformationServiceTest extends AbstractTransformationServiceTest {
```

Comment:
Would be nice to see here a failure test which produces a TransformationException as well.
Maybe you can copy it from my PR: https://github.com/sja/smarthome/commit/003f7218739ec1912bf4151dfe15f8c937a8d12a